pipelines/folioRancher/folioNamespaceTools/manageNamespace/createNamespaceFromPSQLDump/Jenkinsfile

```diff
@@ -0,0 +1,101 @@
+#!groovy
+import org.folio.Constants
+
+@Library('pipelines-shared-library@RANCHER-741-Jenkins-Enhancements') _
+
+CONFIG_BRANCH = 'RANCHER-741-Jenkins-Enhancements'
+import org.folio.models.TerraformConfig
+import org.jenkinsci.plugins.workflow.libs.Library
+
+properties([
+  buildDiscarder(logRotator(numToKeepStr: '20')),
+  parameters([
+    string(name: "CLUSTER", defaultValue: "folio-testing"),
+    string(name: "NAMESPACE", defaultValue: "ecs-snapshot"),
+    string(name: "AGENT", defaultValue: "jenkins-agent-java17"),
```

Comment:
CLUSTER, NAMESPACE, AGENT, BRANCH parameters could be replaced by folioParameters methods